packages/itwinui-css/src/searchbox/searchbox.scss

```diff
@@ -15,24 +15,27 @@
 }
 
 .iui-expandable-searchbox {
-  width: var(--_iui-input-min-height);
-  transition: width var(--iui-duration-1) ease-in-out;
+  // increase specificity to override input-flex-container
+  &#{&} {
```

Comment:
perfectly demonstrates why having one stylesheet is better. previously we were relying on order of imports, `input-container.css` needs to come before `searchbox.css` - this is brittle because user app could additionally import `input-container.css` later.

"correct" solution would be to control the ordering/layering in `all.css`, but for now i'm increasing the specificity here